pkg/logging/logging.go

```diff
@@ -170,3 +171,10 @@ func WithLevelFunctionHook(l *zap.Logger, fn IncrementLogMetricsFunc) *zap.Logge
 
 	return l
 }
+
+// Close closes an object and logs an error in case of failure.
+func Close(ctx context.Context, obj io.Closer, errorMessage string) {
+	if err := obj.Close(); err != nil {
+		FromContext(ctx).Error(errorMessage, zap.Error(err))
+	}
+}
```

Comment:
I'm not sure the logging package is the best place for this. `ioutil` would be ideal but it clashes with the standard library package

Reply:
This is here because it is a Close with logger.
There are no other Close in this package as we expose zap directly.